model/variableSheet.cc

```diff
@@ -0,0 +1,67 @@
+/*
+  @copyright Steve Keen 2018
+  @author Russell Standish
+  This file is part of Minsky.
+
+  Minsky is free software: you can redistribute it and/or modify it
+  under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Minsky is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Minsky.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "variableSheet.h"
+#include "latexMarkup.h"
+#include "selection.h"
+#include <pango.h>
+#include "minsky_epilogue.h"
+using namespace std;
+using ecolab::cairo::Surface;
+using ecolab::Pango;
+using ecolab::cairo::CairoSave;
+
+namespace minsky
+{
+	
+namespace
+{    
+  struct CroppedPango: public Pango
```

Comment:
This class is a candidate for being moved into a common header file

Reply:
True, it is repeated in three different headers. Maybe I can just put it in minsky.h like EquationDisplay?